catalog.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="ru">
+  <head>
+    <meta charset="utf-8">
+    <title>HTML Academy: Техномарт</title>
+  </head>
+  <body>
+
+    <header>
+        <a>
+          <img src="https://via.placeholder.com/350x65" width="150" alt="логотип Техномарт">
+        </a>
+        <input type="search" placeholder="Поиск:">
+        <a href="">Закладки:</a>
+        <a href="">Корзина:</a>
+        <a href="">Оформить заказ</a>
```

Comment:
Тут у нас есть 2 виджета и кнопка "Оформить заказ". Лучше сделать вот так
```suggestion
        <ul>
            <li>
                <a href="">Закладки:</a>
            </li>
            <li>
                <a href="">Корзина:</a>
            </li>
        </ul>
        <a href="">Оформить заказ</a>
```